mcs/class/System/System.IO.Compression/DeflateStream.cs

```diff
@@ -611,6 +611,7 @@ static int WriteZStream (SafeDeflateStreamHandle stream, IntPtr buffer, int leng
 			}
 		}
 #else
+		// FIXME? Names should start "mono"?
```

Comment:
If we're going to do this, it should probably be in a separate PR (though I'm not familiar enough to have a strong opinion on the matter), so I don't see any use to these comments.

Reply:
Comments are to form a memory. A github issue should be another option.